torch/csrc/itt_wrapper.cpp

```diff
@@ -0,0 +1,21 @@
+#include <c10/macros/Export.h>
```

Comment:
Why not just put this in `torch/csrc/profiler/iit.cpp`?

Reply:
Because `TORCH_API` is required in `itt_wrapper.cpp`, which was defined in `c10/macros/Export.h`.